batch/batch/worker/worker.py

```diff
@@ -341,7 +341,7 @@ async def get_container_status(self):
     async def run(self, worker):
         try:
             async with self.step('pulling'):
-                if self.image.startswith('gcr.io/'):
+                if self.image.startswith('gcr.io/') or 'docker.pkg.dev/' in self.image:
```

Comment:
hmm, we really want this to be the host portion of the image not the whole image.

Can we do instead:
```
image_parts = image.split('/')
image_hostname = ''
if len(image_parts) > 1:
    image_hostname = image_parts[0]
if image_hostname == 'gcr.io' or image_hostname.endswith('docker.pkg.dev'):
```

Reply:
Good point! Moved this to a small utility function.